### ISSUE TITLE
fix contact card 

### ISSUE THREAD

**Issue by HRS8** [HRS8/Portfolio#5]
present using email.js but switch to something else and make it work

**Comment by Aditya6702** [comment]
Do you want to try using some API's? but you still need to write the component in js I guess

**Comment by HRS8** [comment]
yes i am already using a api emailjs which is not working do you have any better alternatives

**Comment by Aditya6702** [comment]
what about formspree : https://formspree.io/

**Comment by stelixdev** [comment]
bro i have fixed and opened a pull request please check it out

**Comment by HRS8** [comment]
yea i have just merged it first of all good work.I appreciate you for that but some of the fonts seem over the top to me I will revert some of the fonts that you have changed and in your testing did the smtp js worked coz i have just tested it by sending a messege via the contact i haven't received it anyways smtp js is a great alternative thank you i will work on smtp js. If you wanna contact me then dm me hrs8.me

**Comment by HRS8** [comment]
My god u have modified a lot I should have opened a branch on it I only want vast and peralta fonts used in about.jsx and experience.jsx,remove the rest of them and also underlines and blah to the rest of them, I like it to look professional, I am reverting the commit modify those and create another pull req i will merge it,good work keep it up

**Comment by stelixdev** [comment]
_Thanks bro, it's fine to remove them on professional view. I just want it be look like some `modern colorful dark themed portfolio` , which have more hover effects and a variety of fonts.
Soon I am also going to drop a repository of my `portfolio`. If you can make some good changes in it, it will be great for me. At last **`thanks a lot.`**_

**Comment by stelixdev** [comment]
The message by `smtpjs` in might dropped in your spam folder you can just mark it as `not a spam`. And afterwards it will always be dropped in your main `inbox`

**Comment by HRS8** [comment]
defenitely bro i love to contribute too, i do really love the new looks i just can't vibe the fonts thats it. thak you very much

**Comment by Letscodez** [comment]
Is the contact card working??

**Comment by stelixdev** [comment]
> The message by `smtpjs` in might dropped in your spam folder you can just mark it as `not a spam`. And afterwards it will always be dropped in your main `inbox`

_Is the contact card now **working**??_
